README.md

```diff
@@ -1,77 +1,129 @@
-# cytube-docker
+# sync-docker-compose
+Docker compose for sync by calzoneman
 
-Docker image for CyTube
+https://github.com/calzoneman/sync
 
-*This repository is currently very alpha and a work in progress*
+## How to get started
 
-This image uses an Alpine image as a base along with an integrated MariaDB instance, to create a dockerfile/image that could be used for production deployments. This image results in a running instance of Cytube built from a specified commit. Various parts of the config.yaml template have been replaced with environment variables, to allow deployment to be easily adjusted for production. The following environment variables are set in the dockerfile:
+To run this application first you must clone the repo.
 
-```
-MYSQL_HOST=localhost
-MYSQL_PORT=3306
-MYSQL_DATABASE=cytube3
-MYSQL_USER=cytube3
-MYSQL_PASSWORD=UltraSecretPass
-MYSQL_ROOT_PASSWORD=UltraSecretRootPass
-HTTP_PORT=80
-HTTP=true
-HTTPS_PORT=443
-HTTPS=false
-IO=true
-IO_PORT=1337
-IO_DOMAIN=http://localhost
-ROOT_DOMAIN=localhost
-USE_MINIFY=false
-COOKIE_SECRET=change-me
-SYNC_CRTKEY=null
-SYNC_CRT=null
-SYNC_CRTCA=null
-SYNC_TITLE=Sync
-SYNC_DESCRIPTION="Free, open source synchtube"
-YOUTUBE_KEY=null
-CHANNEL_STORAGE=file
-VIMEO_WORKAROUND=false
-TWITCH_ID=null
-MIXER_ID=null
-GIT_COMMIT=0bc866dbfaba0ef09dc703d6ef261bdf2e70b2ff
-```
+Next you will need to run get_latest_configs.sh in order to automatically pull the latest versions of the config from the sync repo. You may also choose to do this manually. You will need the conf/example and a config.yaml files so that they can be mounted into the container. The script is there to make this easier for you by downloading the necessary files.
```

Comment:
I'm kind of wondering if it makes more sense to add calzoneman/sync as a [git submodule](https://github.blog/2016-02-01-working-with-submodules/) of this repo so that everything gets checked out with one `git clone && git submodule init && git submodule update`

Reply:
Added https://github.com/calzoneman/sync.git as a submodule